warrior/Framework/Utils/data_Utils.py

```diff
@@ -1266,13 +1266,15 @@ def resolve_argument_value_to_get_tag_value(datafile, system_name,
     if element_value_in_argument.startswith("tag"):
         tag_name = evaluate_tc_argument_value(element_value_in_argument)
         if tag_name is not False:
-            system_name_list = xml_Utils.get_matching_firstlevel_children_from_root(datafile, "system")
+            system_name_list = xml_Utils.get_matching_firstlevel_children_from_root(datafile, 
+                                                                                    "system")
             if system_name_list == [] or system_name_list is None or system_name_list is False:
```

Comment:
please see if it good to add a print that system with provided name could not be found hence will return the value without tag= substituions